tests/test_api_client.py

```diff
@@ -615,3 +621,23 @@ async def test_stream_repacks_exception(client, httpx_mock):
         async for w in client._stream("GET", "queues/123/export?format=csv&exported_at=invalid"):
             pass
     assert str(err.value) == "HTTP 404, content: exported_at: Enter a valid date/time"
+
+
+@pytest.mark.asyncio
+async def test_get_token_new(client, login_mock):
+    client.token = None
+    token = await client.get_token()
+    assert token == OUR_TOKEN
+    assert client.token == OUR_TOKEN
+
+
+@pytest.mark.asyncio
+async def test_get_token_old(client):
+    token = await client.get_token()
+    assert token == FAKE_TOKEN
+
+
+@pytest.mark.asyncio
+async def test_get_token_refreshed(client, login_mock):
```

Comment:
Nitpick: Perhaps renaming this test to `test_get_token_force_refresh` would be slightly more descriptive.

Reply:
makes sense :)